lore/estimators/sklearn.py

```diff
@@ -16,20 +16,35 @@
 from lore.util import timed, before_after_callbacks
 
 require(lore.dependencies.SKLEARN)
+import sklearn
 
 
 class Base(lore.estimators.Base):
-    def __init__(self, estimator):
+    def __init__(self, estimator, eval_metric='sklearn_default', scoring_metric='sklearn_default'):
         super(Base, self).__init__()
         self.sklearn = estimator
+        self.eval_metric = eval_metric
+        self.scoring_metric = scoring_metric
+
+    def __setstate__(self, dict):
```

Comment:
@montanalow A quick look over of this section is all that should be needed. Want to make sure I'm providing backwards compatibility properly for pre-0.7 models.

Reply:
👍